src/client/index.tsx

```diff
@@ -6,7 +6,7 @@ import * as Sentry from '@sentry/browser';
 import Vaken from './app';
 import STRINGS from './assets/strings.json';
 
-Sentry.init({ dsn: STRINGS.SENTRY_URL, tracesSampleRate: 1.0 });
+Sentry.init({ dsn: process.env.SENTRY_URL, tracesSampleRate: 1.0 });
```

Comment:
This won't actually work once it gets to production, as process.env doesn't exist on the client side. You can use the [webpack definePlugin](https://webpack.js.org/plugins/define-plugin/) instead to define SENTRY_URL as a build time constant, and process.env _will_ be defined for the webpack.config.js files.

So:
```
// webpack.something.js
new webpack.DefinePlugin({
  "process.env.SENTRY_URL": JSON.stringify(process.env.SENTRY_URL),
  // ALTERNATIVELY:
  SENTRY_URL: JSON.stringify(process.env.SENTRY_URL), // and access with just the global constant SENTRY_URL
});
```

Reply:
_yike_ good catch

Reply:
I like the alternate syntax here bc if we reuse process.env that is sort of misleading in that process.env won't actually be available client-side

Reply:
Worth noting that TS will complain about it being undefined. You can add a globals.d.ts file with `declare var SENTRY_URL: string;` to make it happy

Reply:
Oop my b tyty

Reply:
@leonm1 regarding definePlugin, should I be expecting that the built code in dist has the actual url string in place of SENTRY_URL, or would it still show up in the code as SENTRY_URL?

Reply:
jk nvm